.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+<!--
+
+Thank you for sending a pull request! Here are some tips:
+
+1. To surface this PR in the changelog add the label: changelog
```

Comment:
this is cool, do we do anything with changelog labels yet? edit to add: I just looked at the file below and it seems like we do! So neat I missed that somehow